modules/sbo-deploying-the-spring-petclinic-sample-application-power-z.adoc

```diff
@@ -0,0 +1,67 @@
+[id="sbo-deploying-the-spring-petclinic-sample-application-ibm-power-z_{context}"]
+= Deploying the Spring PetClinic sample application
+
+To deploy the Spring PetClinic sample application on an {product-title} cluster, you must use a deployment configuration and configure your local environment to test the application.
+
+.Procedure
+
+. Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-postgresql`. Click the btn:[+Add] button and select *Import from Git*.
```

Comment:
Being we now have a multi arch image we can use the x86 steps under deploying the spring pet clinic and replace the p/z section with those steps.  Required changes will be Under step 1 we need to change the yaml

1. Deploy the spring-petclinic-rest application with the PostgresCluster custom resource (CR) by running the following command in shell:

```

oc apply -f - << EOD
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: spring-petclinic-rest
  namespace: my-postgresql
  labels:
    app: spring-petclinic-rest
spec:
  replicas: 1
  selector:
    matchLabels:
      app: spring-petclinic-rest
  template:
    metadata:
      labels:
        app: spring-petclinic-rest
    spec:
      containers:
        - name: application
          image: quay.io/service-binding/spring-petclinic-rest:latest
          env:
          - name: SPRING_PROFILES_ACTIVE
            value: postgresql,spring-data-jpa
          - name: org.springframework.cloud.bindings.boot.enable
            value: "true"
          ports:
          - name: http
            containerPort: 9966
      imagePullPolicy: Always
---
apiVersion: v1
kind: Service
metadata:
  name: spring-petclinic-rest
  namespace: my-postgresql
spec:
  ports:
  - port: 80
    targetPort: 9966
  selector:
    app: spring-petclinic-rest
---
EOD

```

Reply:
@johusta was this new image tested on p/z? can you confirm please?

Reply:
@pedjak yeah I have tested that a couple of times with that new multi arch image on Z. @pravin-dsilva is verifying on P. @ktania46 you mentioned you wanted to change the x86 docs to use this image as well. Someone should test on x86 to make sure it works.